jsk_baxter_robot/baxtereus/baxter-softhand-interface.l

```diff
@@ -196,7 +196,13 @@
         (if safe
           (setq *baxter* (instance baxter-robot-safe :init))
           (setq *baxter* (instance baxter-robot :init)))
-        (send *baxter* :angle-vector (send *ri* :state :potentio-vector))))
+        (send *baxter* :angle-vector (send *ri* :state :potentio-vector))
```

Comment:
Sorry for very minor comment, but I think this line should be after translating end-coords.
What do you think?

Reply:
umm, thats right.
I will change it.